### PR TITLE
Add a RHEL7 dockerfile and standarize format

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by paulfantom** [comment]
Sorry, wrong repo :)

/lgtm cancel

**Comment by smarterclayton** [comment]
This may be failing due to binary location.  Trying to enforce consistency so we can auto-check and make sure the official builds are clean, but will look later

**Comment by squeed** [comment]
@smarterclayton is this good to go? seems fine to me.

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-COPY --from=build-env /go/src/github.com/openshift/cluster-network-operator/_output/linux/amd64/cluster-network-operator /bin/cluster-network-operator
-COPY --from=build-env /go/src/github.com/openshift/cluster-network-operator/_output/linux/amd64/cluster-network-renderer /bin/cluster-network-renderer
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/cluster-network-operator/_output/linux/amd64/cluster-network-operator /usr/bin/
+COPY --from=builder /go/src/github.com/openshift/cluster-network-operator/_output/linux/amd64/cluster-network-renderer /usr/bin/
 COPY manifests /manifests
 COPY bindata /bindata
-
 ENV OPERATOR_NAME=cluster-network-operator
 CMD ["/bin/cluster-network-operator"]
```
Comment (squeed) [inline Dockerfile]: probably needs to be `/usr/bin/cluster-network-operator`?

**Comment by smarterclayton** [comment]
Fixed the typo

**Comment by smarterclayton** [comment]
ready to merge, PTAL

**Pull request merged**